controllers/getCategories.controller.js

```diff
@@ -1,12 +1,26 @@
-const { fetchCategories } = require("../models/fetchCategories.model");
+const {
+  fetchCategories,
+  fetchReviewsId,
+} = require("../models/fetchCategories.model");
 
 exports.getCategories = (req, res) => {
-  fetchCategories()
+  fetchCategories().then(({ rows }) => {
+    res.status(200).send({ categories: rows });
+  });
+};
+
+exports.getReviewsId = (req, res) => {
+  const { review_id } = req.params;
+
+  fetchReviewsId(review_id)
     .then(({ rows }) => {
-      res.status(200).send({ categories: rows });
+      res.status(200).send({ reviews: rows[0] });
```

Comment:
i would consider changing this to be `review` instead of `reviews` as it is just returning one single review